app/controllers/api/account.php

```diff
@@ -1770,10 +1769,6 @@
 
         $provider = Authorization::skip(fn () => $dbForProject->getDocument('providers', $providerId));
 
-        if ($provider->isEmpty()) {
-            throw new Exception(Exception::PROVIDER_NOT_FOUND);
-        }
-
```

Comment:
Why aren't we doing any validation on the provider?

Reply:
It's optional here